packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -364,6 +369,13 @@ export class ContainerDefinition extends cdk.Construct {
     this.ulimits.push(...ulimits);
   }
 
+  /**
+   * This method adds one or more container dependencies to the container.
+   */
+  public addContainerDependencies(...containerDependencies: ContainerDependency[]) {
```

Comment:
Please also add an optional array property to the Props object to initialize these.

Reply:
See my comment below, I'm keeping it consistent with similar properties but open to refactoring all the properties to output cleaner cloud formation.